Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -398,7 +398,7 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
         let (project, _) = projectSignal
 
         guard featureFacebookConversionsAPIEnabled(), project.sendMetaCapiEvents == true,
-          let externalId = AppTrackingTransparency.advertisingIdentifier() else { return }
+          let externalId = AppTrackingTransparency().advertisingIdentifier() else { return }
```

Comment:
Since we have it in the app environment, we can use `AppEnvironment.current.appTrackingTransparency` instead of creating a new instance of the struct. The only place we'd need to create that struct is after user is prompted and the dialog returns their answer.